src/components/Maps/InfoWindow.tsx

```diff
@@ -64,6 +64,7 @@ const InfoWindow = ({
       }}
       hideBackdrop
       disableEnforceFocus
+      disableScrollLock
```

Comment:
I'm just guessing that this allows the user to scroll up and down in the InfoWindow, right?

Reply:
`disableScrollLock` allows the user to scroll the background when modals are open. Weirdly enough, if there is a scroll lock, then padding is added